packages/workbox-cli/src/bin.js

```diff
@@ -21,6 +21,16 @@ const meow = require('meow');
 const cliHelpText = require('./cli-help');
 const CLI = require('./index.js');
 
+/* eslint-disable no-console */
+// argv[0] is the path to the node runtime.
+// argv[1] is the path to the script that node is running.
+if (process.argv[1].endsWith('workbox-cli')) {
+  console.warn(`Please run the command line tool as 'workbox' instead of ` +
+    `'workbox-cli'.\n'workbox-cli' will stop working in the next major ` +
+    `release.\n(See: https://github.com/GoogleChrome/workbox/issues/730)`);
```

Comment:
I'd normally err on the side of caution with changes like this (a single major release waiting period vs longer for CLI renames), but given our low usage numbers atm, this sounds fine. I really like the idea of just directly using `workbox` for this.